### PR TITLE
Devx-1063: Deprecation notes

### PR DESCRIPTION
**Pull request by Astro-Choco**
README: Added a deprecation note in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Zettle Go API Beta Documentation
 
+:warning: **Starting May 9, 2022, this repository is deprecated and will no longer be used to make any documentation updates.**
```
Comment (AndreaFilyo) [inline README.md:3]:
```suggestion
:warning: **Starting May 9, 2022, this repository is deprecated and will no longer be updated.** 
```

**Review by AndreaFilyo — APPROVED** [review]
Small comment, good to go.

File: README.md (hunk 1 above)
Comment (Astro-Choco) [reply README.md]: Fixed

**Review by taizamontenegro — APPROVED** [review]
lgtm

**Review by AndreaFilyo — APPROVED** [review]
Good to go

**Pull request merged**